server.go

```diff
@@ -2154,10 +2154,11 @@ func (s *Server) executeShowMeasurementsStatement(stmt *influxql.ShowMeasurement
 		row.Values = append(row.Values, []interface{}{v})
 	}
 
-	// Make a result.
+	// Make result.
 	result := &Result{
-		Rows: influxql.Rows{row},
+		Rows: make(influxql.Rows, 1),
 	}
+	result.Rows[0] = row
```

Comment:
This change was due to go vet failing.  There is an unfortunate side affect of typed slices outside of the package that it doesn't understand, and will likely not change this behavior.